app/assets/stylesheets/layouts/_results.scss

```diff
@@ -19,6 +19,33 @@
   }
 }
 
+.l-results__firm-name {
+  @include column(12);
+
+  margin-left: 0;
+  margin-right: 5%;
+
+  @include respond-to($mq-m) {
+    @include column(8);
+
+    margin-left: 0;
+    margin-right: 5%;
+  }
+}
+
+.l-results__profile-link {
+  @include column(12);
+
+  text-align: right;
+  margin-right: 0;
+
+  @include respond-to($mq-m) {
+    @include column(4);
+
+    margin-right: 0;
+  }
+}
+
 .l-results__heading {
   @include column(12);
 
```

Comment:
```WARNING on line 102 of /Users/danm/Work/cultivate/mas/rad/rad_consumer/app/assets/stylesheets/layouts/_results.scss: ".l-results__filter-button-container" failed to @extend ".l-results-filter".
  The selector ".l-results-filter" was not found.
  This will be an error in future releases of Sass.
  Use "@extend .l-results-filter !optional" if the extend should be able to fail.```

In the Rails server console output.